lib/stoplight/notifier/slack.rb

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Slack < Base
+      # @return [Proc]
+      attr_reader :formatter
+
+      # @return [::Slack::Notifier]
+      attr_reader :slack_notifier
+
+      # @param slack_notifier [::Slack::Notifier]
+      # @param formatter [Proc, nil]
+      def initialize(slack_notifier, formatter = nil)
```

Comment:
I was thinking of passing an already configured `Slack::Notifier` object and then you don't need any options.

You can override those options for any `Slack::Notifier#ping` but I'm not sure it's that useful in Stoplight's usecase.

On the other hand this isn't hard to change so I can modify this PR. What do you think?